crates/runc-shim/README.md

```diff
@@ -25,3 +25,13 @@ $ sudo ctr run --rm --runtime io.containerd.runc.v2-rs -t docker.io/library/hell
 ```
 
 You can run a container by `ctr`, `crictl` or kubernetes API.
+
+## Performance test
+
+testing environment: *16U32G Ubuntu20.04, containerd v1.6.8 + runc v1.1.4*
```

Comment:
Could you possibly add a few more details how benchmarking was performed?

Reply:
done